src/rdb.c

```diff
@@ -1145,6 +1145,8 @@ ssize_t rdbSaveSingleModuleAux(rio *rdb, int when, moduleType *mt) {
     /* Save a module-specific aux value. */
     RedisModuleIO io;
     int retval = rdbSaveType(rdb, RDB_OPCODE_MODULE_AUX);
+    if (retval == -1) return -1;
+    io.bytes += retval;
```

Comment:
just want to confirm that this fix is just something you noticed, right?
didn't cause any real harm?

i.e. the wrongly counted io.bytes doesn't have a real implication, and the next attempt to save something a few lines below would fail too exit with -1

Reply:
Exactly